packages/requests/src/lib/requests-result.spec.ts

```diff
@@ -653,4 +655,41 @@ describe('requests result', () => {
 
     expect(reqSpy).toHaveBeenCalledTimes(3);
   });
+
+  it('should use additional cache keys', () => {
+    jest.useFakeTimers();
+
+    const { state, config } = createState(withEntities<Todo>());
+
+    const store = new Store({ state, config, name: 'todos' });
+
+    const spy = jest.fn();
+
+    function getTodos() {
+      return timer(1000).pipe(
+        map(() => [createTodo(1)]),
+        tap((todos) => store.update(setEntities(todos))),
+        tap(todo => spy(todo)),
+        trackManyRequestResult(['todos'], todos => todos.map(todo => (['todos', todo.id])))
+      );
+    }
+
+    getTodos().subscribe();
+
+    function getTodo(id: number) {
+      return timer(1000).pipe(
+        map(() => createTodo(id)),
+        tap((todo) => store.update(upsertEntities([todo]))),
+        tap(todo => spy(todo)),
+        trackRequestResult(['todos', id])
+      );
+    }
+
+    // Todos must have been fetched before caching can be done.
+    timer(1000).subscribe(() => getTodo(1).subscribe());
```

Comment:
Add a test with joinRequestResult for todo id 1 and check the response and the cached response with the number of times it's called

Reply:
New test has been added, not sure if it covers what you meant though.